recipes/functions-framework-cpp/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+    generators = "cmake", "cmake_find_package_multi", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    short_paths = True
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "9",
+            "Visual Studio": "15.7",
+            "clang": "7",
+            "apple-clang": "11",
+        }
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 17)
+        else:
+            raise ConanInvalidConfiguration(
+                "{} requires C++17. Using the compiler's default will result in link errors against Abseil".format(
+                    self.name
+                )
+            )
+        minimum_version = self._compilers_minimum_version.get(
+            str(self.settings.compiler), False
+        )
+        if minimum_version:
+            if tools.Version(self.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    "{} requires C++17, which your compiler does not support.".format(
+                        self.name
+                    )
+                )
+        else:
+            self.output.warn(
+                "{} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(
+                    self.name
+                )
+            )
+
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("Fails to compile for Windows as a DLL")
+
+        if hasattr(self, "settings_build") and tools.cross_building(self):
+            raise ConanInvalidConfiguration(
+                "Recipe not prepared for cross-building (yet)"
+            )
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            destination=self._source_subfolder,
+            strip_root=True
+        )
+
+    def requirements(self):
+        self.requires("abseil/20210324.2")
+        self.requires("boost/1.75.0")
+        self.requires("nlohmann_json/3.10.2")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["FUNCTIONS_FRAMEWORK_CPP_TEST_EXAMPLES"] = False
+
+        self._cmake.configure()
+        return self._cmake
+
+    def _patch_sources(self):
+        pass
+
+    def build(self):
+        self._patch_sources()
```

Comment:
```suggestion
    def build(self):
```

_patch_sources can be removed

Reply:
Fixed locally, thanks.  I will wait until the previous comment is resolved before pushing the change, no need to waste build resources.